hedera-mirror-rest/api/v1/openapi.yml

```
@@ -2011,6 +2016,23 @@ components:
           format: int64
           nullable: true
           type: integer
+        block_gas_used:
+          description: The total amount of gas used in the block
+          example: 2000
+          format: int64
+          nullable: true
+          type: integer
+        block_hash:
+          description: The hex encoded block (record file chain) hash
+          example: "0x6ceecd8bb224da491"
+          nullable: true
+          type: string
+        block_number:
+          description: The block number. Since Hedera does not have the native concept of blocks, this counts the number of record files seen since the mirror node's configured start date. This can vary between mirror nodes that use different start dates.
```

Guess we should update this in both places as this description is no longer accurate.

```suggestion
          description: The block height calculated as the number of record files  starting from zero since network start.
```